client/my-sites/stats/activity-log/index.jsx

```diff
@@ -375,6 +386,7 @@ class ActivityLog extends Component {
 				<QuerySiteSettings siteId={ siteId } />
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
+				{ this.renderUpgradeBanner() }
```

Comment:
there's no need or real benefit to having a separate method here…we can just inline the check

```js
{ this.props.siteIsOnFreePlan && <UpgradeBanner /> }
```